jekyll/_cci2_ja/runner-installation-linux.adoc

```diff
@@ -246,3 +246,8 @@ circleci.service - CircleCI Runner
 ```shell
 journalctl -u circleci
 ```
+
+[#troubleshooting]
+== トラブルシューティング
+
+Refer to the <<troubleshoot-self-hosted-runner#troubleshoot-machine-runner, Troubleshoot Machine Runner section>> of the Troubleshoot Self-hosted Runner guide if you encounter issues installing or running machine runner on Linux.
```

Comment:
Hello, @Mayuko-Sa ! Do we expect this text to also be translated within this PR?